deploy/helm/templates/logs-fargate-config-map.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if and .Values.fargate.enabled .Values.fargate.logs.enabled }}
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+{{ include "common.labels" . | indent 4 }}
+---
+{{- $pattern := "[^a-zA-Z0-9\\.\\-_/#]" -}}
+{{- $replacement := "" -}}
+{{- $name := regexReplaceAll $pattern .Values.cluster.name $replacement | default .Values.cluster.uid | lower | trunc 512 -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+  labels:
+{{ include "common.labels" . | indent 4 }}
+data:
+  flb_log_cw: "false"  # Set to true to ship Fluent Bit process logs to CloudWatch.
+  filters.conf: |
+    [FILTER]
+        Name parser
+        Match *
+        Key_name log
+        Parser crio
+    [FILTER]
+        Name kubernetes
+        Match kube.*
+        Merge_Log On
+        Keep_Log Off
+        Buffer_Size 0
+        Kube_Meta_Cache_TTL 300s
+        Labels Off
+        Annotations Off
+{{- if .Values.fargate.logs.filters }}
+{{ toString .Values.fargate.logs.filters | indent 4 }}
+{{- end }}
+    [FILTER]
+        Name modify
+        Match *
+        Add sw.k8s.cluster.uid {{ .Values.cluster.uid }}
+        Add sw.k8s.log.type container
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch_logs
+        Match kube.*
+        region {{ .Values.fargate.logs.region }}
```

Comment:
We had that discussion a few days ago and the outcome was that we do the same thing for `cluster.name`, `cluster.uid` and `otel.endpoint`. Do we want to apply such checking everywhere?

Reply:
No I don't want to apply the same thing everywhere. I wasn't aware of `required` option though, I realized that option today. The issue isn't that high on other places because there it will lead to Pod startup failures or logs in error. What will be result of missing region in this configmap? Therefore I would apply this only to this new region

Reply:
changed